node/service/src/lib.rs

```diff
@@ -130,7 +131,7 @@ fn new_partial<RuntimeApi, ExecutorDispatch>(
 		FullClient<RuntimeApi, ExecutorDispatch>,
 		FullBackend,
 		FullSelectChain,
-		sc_consensus::DefaultImportQueue<Block, FullClient<RuntimeApi, ExecutorDispatch>>,
```

Comment:
This change comes from substrate [PR #14612](https://github.com/paritytech/substrate/pull/14612)